README.md

```diff
@@ -695,13 +701,18 @@ into the VM-s.
 On the coordinator node:
 
 ```bash
-# an example usage: Use PostgreSQL 12.1 and run valgrind test on enterprise/enterprise-master
-fab use.postgres:12.1 use.enterprise:enterprise-master run.valgrind
+# an example usage: Use PostgreSQL 14.2 and run valgrind test on enterprise/enterprise-master
+fab use.postgres:14.2 use.enterprise:enterprise-master run.valgrind
 ```
```

Comment:
Either update this with the run.valgrind:<specific_schedule> and appropriate comment
OR remove it completely, and update the README to say smth like: 
`The command can be run without tmux, but since the valgrind test run for too long we prefer to run in a detached session with tmux`.

Reply:
> The command can be run without tmux, but since the valgrind test run for too long we prefer to run in a detached session with tmux

This is already mentioned in a few lines above:
```
However as valgrind tests take too much time to complete, we recommend you to run valgrind tests in a detached session:
```

And given that this is just an example, I usually avoid updating `12.1` but your suggestion makes sense.